src/core/plugins/json-schema-2020-12-samples/fn/api/optionAPI.js

```diff
@@ -13,5 +13,7 @@ const optionAPI = (optionName, optionValue) => {
 
   return registry.get(optionName)
 }
+optionAPI.getDefaults = () => registry.defaults
+optionAPI.setDefaults = () => registry.registerMany(registry.defaults)
```

Comment:
Why doing this? All the options are already set when OptionsRegistry is create, right?

Reply:
Yeah but what if you change some and then want to set it back to defaults. I'm using this in the unit tests where we start every test from the defaults.